content/05.discussion.md

```diff
@@ -1,20 +1,21 @@
 ## Discussion
 To our knowledge, TPOT-DS is the first AutoML tool to offer the option of feature selection at the group level.
 Previously, it was computationally expensive for any AutoML program to process biomedical big data.
-TPOT-DS is able to identify the most meaningful group of features to include in the prediction pipeline. 
+TPOT-DS is able to identify the most meaningful group of features to include in the prediction pipeline.
 We assessed TPOT-DS's out-of-sample prediction accuracy compared to standard TPOT and XGBoost, another state-of-the-art machine learning method.
 We applied TPOT-DS to real-world expression data to demonstrate the identification of biologically relevant groups of genes.
 
-Implemented with a strongly typed GP, Template allows users to pre-specify a particular pipeline structure, which speeds up the automation computation time and provides potentially more interpretable results.
-Consequently, Template enables the comparison between the two TPOT implementations, with and without DS.
+Implemented with a strongly typed GP, Template provides more flexibility by allowing users to pre-specify a particular pipeline structure based on their knowledge, which speeds up AutoML process and provides potentially more interpretable results.
+For example, for high-dimensional data, dimension reduction algorithms or feature selection algorithms are preferred to add into pipeline via Template for reducing computation time and potentially detecting important features.
```

Comment:
For example, in high-dimensional data, dimensionality reduction or feature selection algorithms are preferably included at the beginning of the pipelines via Template to  identify important features and, meanwhile, reduce computation time.